PlayCover/Model/PlayApp.swift

```diff
@@ -35,9 +35,13 @@ class PlayApp: BaseApp {
     func launch() async {
         do {
             isStarting = true
+            let noMacAlert = UserDefaults.standard.bool(forKey: "\(info.bundleIdentifier).noMacAlert")
             if prohibitedToPlay {
                 await clearAllCache()
                 throw PlayCoverError.appProhibited
+            } else if hasMacVersion && !noMacAlert, await !runMacOSWarning() {
```

Comment:
in my opinion it is better at startup because it works if you already have an app installed or if an app will be added to the list. but if you want I'll change it

Reply:
I agree with Moon, I would make it install time only.

Reply:
Currently, it seems that you added it to the Downloader instead of the Installer, which gives the incorrect behaviour.